products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
Could be either, not really sure.

On Thu, Jun 16, 2016 at 12:47 PM, rctrh <notifications@github.com> wrote:

> In products/developertoolset/get-started-rhel7-cpp.adoc
> <https://github.com/redhat-developer/developers.redhat.com/pull/1172#discussion_r67402306>
> :
>
> > @@ -1,7 +1,15 @@
> >  :awestruct-layout: product-get-started-multipath
> >  :awestruct-interpolate: true
> > -:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
> > -:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
> > +:tthw-site-base-url: #{site.base_url}
> > +:tthw-cdn-site-base-url: #{cdn(site.base_url)}
>
> So is it that or this related bit which is spitting out the error
> 4 /usr/local/bin/bundle:23:in `
> '
> 13:33:24 Error cdn-ifying img [image: Eclipse CPP Logo]
> 13:33:24 While processing file /home/awestruct/
> developer.redhat.com/products/developertoolset/get-started-rhel6-cpp.adoc
> 13:33:24 An error occurred: Is a directory @ io_fread - /home/awestruct/
> developer.redhat.com
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/1172/files/67c91b52a73fc9ea11054e760264c80ab10704f0#r67402306>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAB1E-MeT2ygZRISB225_mWdyEG6Tt1xks5qMZovgaJpZM4I1okA>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp